### PR TITLE
Chip documentation/tas 303

### PR DESCRIPTION
**Pull request by visagim**
- [x] changelog

### CODE REVIEW

**Comment by visagim** [comment]
I think I just forgot to push the last changes I had (at the `__init__.py` with most of the explanation for the `chip` module). Let me know if there are any classes with missing documentation

File: src/qililab/chip/chip.py [hunk 1]
```diff
@@ -24,7 +27,10 @@
 
 @dataclass
 class Chip:
-    """Chip representation as a graph."""
+    """Chip representation as a graph.
+    This class represents the chip structure of in the runcard and contains all the connectivity information of
+    the chip.
+    """
 
     nodes: list[Node]
```
Comment (jjmartinezQT) [inline src/qililab/chip/chip.py:35]: This attribute has no definition in the documentation. Could you include it, please?

File: src/qililab/chip/__init__.py [hunk 2]
```diff
@@ -13,7 +13,24 @@
 # limitations under the License.
 
 """
-This module contains the classes needed to define a quantum chip.
+The chip class contains information about the chip and its conections. This information can be reached through the Platform class through a platform object by accessing platform.chip.
+
+Each node in the chip can be a Coil, Coupler, Port, Qubit or Resonator. Each node also has an alias assigned to it. For example, the alias for Qubit nodes is typically ``qubit_n`` where n is the qubit index.
+
+The qubit connectivity (chip topology) can be accessed by calling chip.get_topology(), which returns a networkx graph of the qubits Eg.
```
Comment (jjmartinezQT) [inline src/qililab/chip/__init__.py:20]: the abbreviated of 'for example' is written in lower case and separated by dots in between, often with a comma before it to separate it from the rest of the sentence. I would rewrite this like:

"The qubit connectivity (chip topology) can be accessed by calling chip.get_topology(), which returns a networkx graph of the qubits, e.g."

File: src/qililab/chip/node.py [hunk 3]
```diff
@@ -21,7 +21,9 @@
 
 @dataclass(kw_only=True)
 class Node(Settings, FactoryElement):
-    """Class representing a node of the chip's graph."""
+    """This class is used to represent a node of the chip's graph.
+    Each node is an element of the chip
+    """
 
     alias: str
```
Comment (jjmartinezQT) [inline src/qililab/chip/node.py:28]: This attributes are missing definitions in the docs.

Comment (jjmartinezQT) [reply src/qililab/chip/node.py:28]: Same with Port class and its attributes, Qubit, Resonator, Coupler and Coil. Also this classes could benefit from better definition of what the class is about.

**Review by jjmartinezQT — CHANGES_REQUESTED** [review]

File: src/qililab/chip/chip.py (hunk 1 above)
Comment (jjmartinezQT) [reply src/qililab/chip/chip.py:35]: Addressed.

File: src/qililab/chip/__init__.py (hunk 2 above)
Comment (jjmartinezQT) [reply src/qililab/chip/__init__.py]: done.

File: src/qililab/chip/node.py (hunk 3 above)
Comment (jjmartinezQT) [reply src/qililab/chip/node.py]: First pass at this done

**Review by jjmartinezQT — APPROVED** [review]

**Pull request merged**